File: tools/excel/src/main/java/top/bettercode/summer/tools/excel/ExcelConverter.java
package top.bettercode.summer.tools.excel;

import java.io.Serializable;

/**
 * 转换器
 *
 * @param <F> 源
 * @param <T> 目标
 */
@FunctionalInterface
public interface ExcelConverter<F, T> extends Serializable {

  T convert(F from);
}

File: tools/excel/src/main/kotlin/top/bettercode/summer/tools/excel/ExcelConverter.kt
package top.bettercode.summer.tools.excel

import java.io.Serializable

/**
 * 转换器
 *
 * @param <F> 源
 * @param <T> 目标
</T></F> */
@FunctionalInterface
fun interface ExcelConverter<F, T> : Serializable {
    fun convert(from: F): T
}